bundles/org.openhab.binding.groheondus/src/main/java/org/openhab/binding/groheondus/internal/handler/GroheOndusSenseHandler.java

```diff
@@ -103,16 +117,16 @@ protected Measurement getLastDataPoint(Appliance appliance) {
             return new Measurement();
         }
         List<Measurement> measurementList = applianceData.getData().getMeasurement();
-
+        Collections.sort(measurementList, Comparator.comparing(e -> ZonedDateTime.parse(e.timestamp)));
```

Comment:
Is this really needed? If I remember correctly, the datapoints in the API are already returned in an ordered list 🤔

Reply:
Possibly not, I added it while I did debug why the data returned was old (which in fact happened due to the timestamp -> date conversion in the API package). 

I would argue that the parsing of Strings->Date objects should have been done in the API package. Agreed?